exercises/isogram/isogram_test.rb

```diff
@@ -7,55 +7,55 @@ class IsogramTest < Minitest::Test
   def test_empty_string
     # skip
     string = ""
-    assert Isogram.is_isogram?(string)
+    assert Isogram.isogram?(string), "Expected 'true', '' is an isogram"
```

Comment:
I am not sure if true/false should be quoted. I have a slight preference for not quoting it.
@kotp?